controllers/ucs_controller.py

```diff
@@ -221,7 +240,10 @@ def getServiceProfile(host=None, user=None, password=None):
 
 
 @http_body_factory
-def powerMgmt(host=None, user=None, password=None, identifier=None, action=None):
+def powerMgmt(identifier=None, action=None):
+    host = request.headers.get('ucs-host')
+    user = request.headers.get('ucs-user')
+    password = request.headers.get('ucs-password')
```

Comment:
Considering there is a lot copies for above 3 lines of code, create a function for them will remove these duplicated code and conform to the clean code practice: https://github.com/ryanmcdermott/clean-code-javascript#remove-duplicate-code